src/GenreSelect/GenreSelect.js

```diff
@@ -0,0 +1,26 @@
+import react from "react";
+
+function GenreSelect(props) {
+  const Genres = ['Action', 'Comedy', 'Drama', 'Fantasy']
```

Comment:
By convention, JavaScript variable names are written in camelCase

Reply:
fixed